### DISCUSSION TITLE
Is upgrade to django 4.2 (LTS) or 5.0 planned in the near future?

### DISCUSSION THREAD

**Discussion by bartacc** [silviogutierrez/reactivated#329] [Q&A] [answered]
Hi,
are there plans to update reactivated to django 4.2 or 5.0?

I've been reading about this library and I really like your approach. Using react this way would be perfect for my use case. I'm only worried that currently it supports only django 4.0. It would be really nice to use it at least with the Long Term Release of Django (4.2). Or maybe it's already compatible?

Anyway, thanks for the good work, library like this one is really needed!

**Comment by silviogutierrez** [comment, marked as answer]
Yes, the plan is to move to Django 5.0 and Python 3.12 very soon.

**Reply by bartacc** [reply]
Awesome, thanks for the answer.
